package.json

```diff
@@ -43,5 +43,12 @@
     "npm": "2.14.4"
   },
   "author": "Anthony Garvan",
-  "license": "CC0"
+  "license": "CC0",
+  "devDependencies": {
+    "eslint": "^2.9.0",
+    "eslint-config-airbnb": "^9.0.1",
+    "eslint-plugin-import": "^1.7.0",
+    "eslint-plugin-jsx-a11y": "^1.2.0",
+    "eslint-plugin-react": "^5.1.1"
```

Comment:
Not strictly necessary, but they are peerDependencies of `estlint-config-airbnb`, so npm will complain if they are not there 😒 :

![image](https://cloud.githubusercontent.com/assets/697848/15197084/250b6620-1795-11e6-890c-57c2ce55173e.png)